static/js/settings_streams.js

```diff
@@ -53,6 +53,8 @@ exports.build_default_stream_table = function (streams_data) {
         },
     }).init();
 
+    ui.set_up_scrollbar(table.closest(".progressive-table-wrapper"));
```

Comment:
Do we need to have the scrollbar update when we add/remove a default stream?  It's minor since they tend to change 1 at a time, but we do, right?